lib/read/lib/Cardano/Wallet/Read/Hash.hs

```diff
@@ -0,0 +1,38 @@
+{- |
+Copyright: © 2024 Cardano Foundation
+License: Apache-2.0
+
+Abstract and specific Hash functionality.
+-}
+module Cardano.Wallet.Read.Hash
+    ( -- * Core operations
+      H.Hash
+    , H.HashAlgorithm (H.digest, H.hashAlgorithmName)
+    , H.sizeHash
+    , H.hashWith
+
+    -- * Conversions
+    , H.castHash
+    , H.hashToBytes
+    , H.hashFromBytes
+    , H.hashToBytesShort
+    , H.hashFromBytesShort
+
+    -- * Rendering and parsing
+    , H.hashToBytesAsHex
+    , H.hashFromBytesAsHex
+    , H.hashToTextAsHex
+    , H.hashFromTextAsHex
+    , H.hashToStringAsHex
+    , H.hashFromStringAsHex
+
+    -- * Specific Hash algorithms
+    , Blake2b_224
+    , Blake2b_256
+    ) where
+
+import Cardano.Crypto.Hash.Blake2b
+    ( Blake2b_224
+    , Blake2b_256
```

Comment:
what about exporting from here https://github.com/cardano-foundation/cardano-wallet/blob/master/lib/crypto-primitives/src/Cryptography/Hash/Blake.hs ?

Reply:
I sympathize, but the problem is that `Cardano.Crypto.Hash.Blake2b` does not re-export from `cryptonite` — rather, it defines a distinct type `Blake2b_256` here:

https://github.com/IntersectMBO/cardano-base/blob/4e1d652343b48d3f02f05601e059b87965504e13/cardano-crypto-class/src/Cardano/Crypto/Hash/Blake2b.hs#L24

We will have to unify / merge in re-export the APIs at some point. At the moment, I think that the high-level API design of the `cardano-crypto-classes` is superior to `cryptonite`, but it covers only those algorithms and use cases that are needed by Cardano.